api/app/Console/Jobs/RandomizeTextJob.php

```diff
@@ -1,181 +1,203 @@
-use App\Helpers\SpreadsheetHelper;
-use App\Models\Table;
-use App\Services\Interfaces\ISpintaxService;
-use App\Services\Interfaces\ISpreadsheetClientService;
-
-class RandomizeTextJob extends JobBase
-{
-    private ISpintaxService $spintaxService;
-
-    /**
-     * @var bool is logging enabled.
-     */
-    protected bool $loggingEnabled = false;
-
-    protected int $maxJobTime = 60*60;
-
-    private XmlGeneration $xmlGeneration;
-
-    /**
-     * Randomises text in specified result column based on pattern column and updates spreadsheet.
-     *
-     * @param string $tableId spreadsheet id.
-     * @param int $patternCol column to take pattern from.
-     * @param int $resultCol column to fill in randomized result.
-     * @param int $numRow row number, required for spreadsheet update.
-     * @param string $sheetName sheet name, required for spreadsheet update.
-     * @param array $row row data.
-     * @param string $quotaUser quota user.
-     */
-    private function randomizeText(
-        string $tableId,
-        int $patternCol,
-        int $resultCol,
-        int $numRow,
-        array $row,
-        string $sheetName,
-        string $quotaUser) : void
-    {
-        $alreadyFilled = isset($row[$resultCol]) && $row[$resultCol] != '';
-        $noSource = !isset($row[$patternCol]) || $row[$patternCol] == '';
-
-        if($alreadyFilled || $noSource)
+    
+    
+    namespace App\Console\Jobs;
+    
+    
+    use App\Configuration\XmlGeneration;
+    use App\Helpers\SpreadsheetHelper;
+    use App\Models\Table;
+    use App\Services\Interfaces\ISpintaxService;
+    use App\Services\Interfaces\ISpreadsheetClientService;
+    use Illuminate\Support\Facades\Log;
+    
+    class RandomizeTextJob extends JobBase {
+        private ISpintaxService $spintaxService;
+        
+        /**
+         * @var bool is logging enabled.
+         */
+        protected bool $loggingEnabled = true;
+        
+        protected bool $timeoutEnabled = false;
+        
+        protected int $maxJobTime = 60 * 60;
+        
+        private XmlGeneration $xmlGeneration;
+        
+        /**
+         * Randomises text in specified result column based on pattern column and updates spreadsheet.
+         *
+         * @param string $tableId spreadsheet id.
+         * @param int $patternCol column to take pattern from.
+         * @param int $resultCol column to fill in randomized result.
+         * @param int $numRow row number, required for spreadsheet update.
+         * @param string $sheetName sheet name, required for spreadsheet update.
+         * @param array $row row data.
+         * @param string $quotaUser quota user.
+         * @throws \Exception
+         */
+        private function randomizeText(
+            string $tableId,
+            int $patternCol,
+            int $resultCol,
+            int $numRow,
+            array $row,
+            string $sheetName,
+            string $quotaUser): void
```

Comment:
Why did you keep this argument if it is not used anywhere?

Reply:
removed